Traceback (most recent call last):
  File "./10.10.redirect_stderr.py", line 7, in <module>
    raise Exception, 'this error will be logged'
Exception: this error will be logged
